packages/vue/src/components/atoms/SfColor/SfColor.html

```diff
@@ -0,0 +1,6 @@
+<button v-on="$listeners" class="sf-color" :class="{ 'sf-color--active' : isActive }" :aria-label="ariaLabel">
```

Comment:
maybe we should also add `aria-pressed` here ? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
Added